libkbfs/conflict_resolver.go

```diff
@@ -910,8 +910,9 @@ func (cr *ConflictResolver) resolveMergedPaths(ctx context.Context,
 	// cache for the merged branch.
 	mergedNodeCache := newNodeCacheStandard(cr.fbo.folderBranch)
 	// Initialize the root node.  There will always be at least one
-	// unmerged path.
-	mergedNodeCache.GetOrCreate(mergedChains.mostRecentMD.data.Dir.BlockPointer,
+	// unmerged path.  Hold it here so it doesn't get gc'd.
+	rootNode, _ := mergedNodeCache.GetOrCreate(
```

Comment:
That's a good point.  I don't think there's a good reason, except that when it's called from `folderBranchOps()`, we expect the root node to already exist (because we only call it when we know another node already exists, which means implicitly the root node must exist due to backpointers).  So doing `GetOrCreate()` could mask errors there, though I don't think it's a big deal.

We wouldn't even have to pass anything extra in, because you can extract the path from the MD that's already being passed in.  Do you think that's a better way?